easybuild/easyconfigs/s/Salmon/Salmon-1.2.1-gompi-2019b.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     GLPv2
+#
+# Notes::
+##
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '1.2.1'
```

Comment:
So, perhaps the biggest question, do we need another version for 2019b? Perhaps squeeze it in for 2020a instead?

Reply:
I am happy to have that in 2020a which is the development if I recall correctly. 
I am always hesitating to use tool-chains which are in development simply I am too remote from what is going on. 
Salmon 1.2.1 was released 22 days ago, according to GitHub. I don't think we got a 1.2.0 ES (released 10.4.2020). Shall I add the 1.2.0 for the 2019b version and get 1.2.1 in for 2020a? How about that?
By the way, why did one of the builds crash? I cannot work out that. 
Thanks!

Reply:
As I mentioned in the review comment, there is a 1.2.0 in 2019b already, and it's quite different from this one (many changes to builddeps)

Reply:
Ok, now I understand. Squeeze it into 2020a and base it on the 1.2.0 version or drop it altogether?

Reply:
Sure, if that is more correct (I really don't know, they are quite a few unusual builddependencies in 1.2.0, but at the same time, I would hope whoever added them had good reasons)